### PR TITLE
Add basic calculator helpers

### PR DESCRIPTION
**Pull request by taroserigano777**
Adds `divide` and `add` helpers.

Open question: should `divide` raise a custom exception on zero?

### CODE REVIEW

File: src/calc.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def divide(a, b):
+    # FIXME: no zero check, will explode at runtime
```
Comment (taroserigano777) [inline src/calc.py:2]: **[blocking]** Division by zero is not handled. This will raise ZeroDivisionError at runtime. Either add a check and raise a custom exception (as mentioned in the PR description) or at minimum validate b != 0.

File: src/calc.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    return a / b
+
+
+def add(a, b):
+    return a + b
+
+
+# TODO: add tests
```
Comment (taroserigano777) [inline src/calc.py:10]: **[blocking]** No tests exist for these functions. Add unit tests covering normal cases, edge cases (zero division, negative numbers, floats), and type errors before merging.

File: src/calc.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+def divide(a, b):
```
Comment (taroserigano777) [inline src/calc.py:1]: **[important]** Missing type hints. Add them for better IDE support and runtime validation: `def divide(a: float, b: float) -> float:`

Comment (taroserigano777) [inline src/calc.py:1]: **[important]** No docstrings. Add brief documentation explaining parameters, return values, and any exceptions raised.

File: src/calc.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    # FIXME: no zero check, will explode at runtime
+    return a / b
+
+
+def add(a, b):
```
Comment (taroserigano777) [inline src/calc.py:6]: **[nit]** The `add` function is trivial and may not warrant its own helper unless there's a plan to extend it with validation or special behavior.

**Review by taroserigano777 — COMMENTED** [review]
## Automated Review Summary

This PR introduces two basic arithmetic functions: `add` and `divide`. The implementation is minimal with no input validation, type hints, or tests.

**Risk:** high — No tests, no input validation, acknowledged division-by-zero bug in production code, and no type safety. The code will fail at runtime with invalid inputs.

### Themes
- Missing test coverage for new functionality
- No input validation or error handling
- Lack of type hints and documentation
- Known bug acknowledged but not fixed

### Findings
- **[blocking]** `src/calc.py` L2 — Division by zero is not handled. This will raise ZeroDivisionError at runtime. Either add a check and raise a custom exception (as mentioned in the PR description) or at minimum validate b != 0.
- **[blocking]** `src/calc.py` L10 — No tests exist for these functions. Add unit tests covering normal cases, edge cases (zero division, negative numbers, floats), and type errors before merging.
- **[important]** `src/calc.py` L1 — Missing type hints. Add them for better IDE support and runtime validation: `def divide(a: float, b: float) -> float:`
- **[important]** `src/calc.py` L1 — No docstrings. Add brief documentation explaining parameters, return values, and any exceptions raised.
- **[nit]** `src/calc.py` L6 — The `add` function is trivial and may not warrant its own helper unless there's a plan to extend it with validation or special behavior.

_— posted by your friendly Agentic DevOps Copilot 🤖_
